tsconfig-common.json

```diff
@@ -17,7 +17,7 @@
         /* Type Checking */
         "strict": true,
         "strictNullChecks": true,
-        "strictFunctionTypes": false,
+        "strictFunctionTypes": true,
```

Comment:
Do we know what the counter argument to this is? Like why would we have turned it off?

Also I think it's true by default so maybe we don't need this line (unless we want to explicitly list it).

Reply:
I don't know if there's any counterargument beyond "you get fewer type errors if you do less rigorous typechecking." IMO TS's strict settings should _always_ be enabled whenever possible.

https://www.typescriptlang.org/tsconfig/#strictFunctionTypes